src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -46,7 +55,7 @@ public Drivetrain() {
   public void periodic(){
     // TODO 2.2.5: Update odometry
 
-    // TODO 1.2.2: Call tankDrive()
+    tankDrive(Robot.driver.getRawLeftTranslation() * .25, Robot.driver.getRawRightTranslation() * .25);
 
```

Comment:
This works, but you might want to move the *.25 to the tankDrive method instead of here so it works in the next activity.

Reply:
Yeah it's probably better to multiple by .25 when you're setting the motor power in the `tankDrive` method.

Reply:
fixed